amanuensis/resources/admin/admin_state.py

```diff
@@ -15,46 +20,55 @@
     "get_all_states",
     "update_project_state",
     "create_consortium",
-    "get_by_code"
+    "get_by_code",
 ]
 
 
 def create_state(name, code):
-    with flask.current_app.db.session as session:  
-        state_schema = StateSchema()  
+    with flask.current_app.db.session as session:
+        state_schema = StateSchema()
         state = udm.create_state(session, name, code)
         state_schema.dump(state)
         return state
 
 
 def get_all_states():
-    with flask.current_app.db.session as session:    
+    with flask.current_app.db.session as session:
         return udm.get_all_states(session)
 
+
 def get_by_code(code):
-    with flask.current_app.db.session as session:    
+    with flask.current_app.db.session as session:
         return udm.get_state_by_code(session, code)
 
 
 def update_project_state(project_id, state_id):
-    with flask.current_app.db.session as session: 
+    with flask.current_app.db.session as session:
         requests = udm.get_requests_by_project_id(session, project_id)
         if not requests:
-            raise NotFound("There are no requests associated to this project or there is no project. id: {}".format(project_id))
+            raise NotFound(
+                "There are no requests associated to this project or there is no project. id: {}".format(
+                    project_id
+                )
+            )
 
         state = udm.get_state_by_id(session, state_id)
         if not state:
             raise NotFound("The state with id {} has not been found".format(state_id))
 
-        request_schema = RequestSchema(many=True)  
-        requests = udm.update_project_state(session, requests, state)
-        request_schema.dump(requests) 
+        request_schema = RequestSchema(many=True)
+        consortium = requests[0].consortium_data_contributor.code
```

Comment:
Request are for different consortiums, so getting only the first one will use the states of the first consortium only.